packages/connect/CHANGELOG.md

```diff
@@ -10,6 +10,23 @@
 
 Use the persistent link [connect.trezor.io/9](https://connect.trezor.io/9/) to access the latest stable version of Connect Explorer.
 
+# 9.4.6-beta.1
+
+This release adds apiType to transports and blocks to blockchainUnsubscribe.
+
+## Feature
+
+-   add apiType to transports (f057f33)
+-   add blocks to blockchainUnsubscribe (de01c1f)
+
+## Fixes
+
+-   move security checks outside getFeatures (e67810d)
```

Comment:
this one on the other hand is interesting and it could fix some of the issues with device loading and communication in general.